### ISSUE TITLE
Unread messages count badge shows "0"

### ISSUE THREAD

**Issue by azzimuth** [zhukov/webogram#1273]
I don't know exactly how to reproduce this but it happened couple of times today.
![webogram_bug1](https://cloud.githubusercontent.com/assets/1640148/20481108/d203d95a-afe5-11e6-8c8a-4ef352f9e14c.png)

**Comment by zhukov** [comment]
yep, I've see this, but have no idea why it happens, the code is straightforward:

```html
<span
      class="im_dialog_badge badge ng-hide"
      ng-show="dialogMessage.unreadCount > 0 && !dialogMessage.pFlags.out"
      ng-bind="dialogMessage.unreadCount"
      my-peer-muted="dialogMessage.peerID" muted-class="im_dialog_badge_muted"
    ></span>
```

Could be possible to debug with steps to reproduce.

**Comment by azzimuth** [comment]
@zhukov I was able to reproduce it today. I had two computers with Webogram open on both with the same account logged in. Then when I got a new message and opened it on one of the computers, when I went to the another and opened the tab in Chrome with Webogram, I had unread messages counter still showing "1". I then scrolled down to the latest message but the badge showed "0".
